scripts/dockerrun.sh

```diff
@@ -136,5 +136,8 @@ docker run $INTERACTIVE -t --rm --sig-proxy=true \
     -e AZUREACCOUNTNAME \
     -e RELEASETOOLSGITURL \
     -e CORESETUPBLOBROOTURL \
+    -e PB_AssetRootUrl \
+    -e PB_PackageVersionPropsUrl \
+    -e PB_PublishBlobFeedUrl \
```

Comment:
Pretty sure these should be uppercased, since [VSTS converts](https://docs.microsoft.com/en-us/vsts/build-release/concepts/definitions/build/variables) when putting variables in the environment.